### ISSUE TITLE
Unable to override the Shuffle and Repeat Drawables

### ISSUE THREAD

**Issue by RowlandOti** [google/ExoPlayer#6709]
### [REQUIRED] Issue description
This is an attempt to override the toggle on, off for both shufle and repeat drawables. It seems to not be working since I still see the default drawables for the two items above,. 

### [REQUIRED] Reproduction steps
Tried overriding the drawables using:
In styles.xml
```xml
</resources>
    <drawable name="exo_controls_shuffle">@drawable/ic_shuffle</drawable>
    <drawable name="exo_controls_shuffle_off">@drawable/ic_shuffle</drawable>
    <drawable name="exo_controls_shuffle_on">@drawable/ic_shuffle</drawable>
    <drawable name="exo_controls_repeat_off">@drawable/ic_repeat</drawable>
    <drawable name="exo_controls_repeat_one">@drawable/ic_repeat</drawable>
    <drawable name="exo_controls_repeat_all">@drawable/ic_repeat</drawable>
</resources>
```

### [REQUIRED] Link to test content
N/A

### [REQUIRED] A full bug report captured from the device
![device-2019-11-29-110123](https://user-images.githubusercontent.com/8356008/69853338-ad97f480-1297-11ea-8fdb-098003c6d1c9.png)

### [REQUIRED] Version of ExoPlayer being used
vERSION IS 2.10.8

### [REQUIRED] Device(s) and version(s) of Android being used
Android API 16 TO API 29

**Comment by ojw28** [comment]
Thanks for reporting this. We should add a layer of indirection to these drawables via [drawables.xml](https://github.com/google/ExoPlayer/blob/r2.10.8/library/ui/src/main/res/values/drawables.xml#L17), like we do for other icons, to make them easier to override.

**Comment by RowlandOti** [comment]
Hoping this will make it in the next release, right?

**Comment by ojw28** [comment]
This will be included in 2.11
